anyrun/src/main.rs

```diff
@@ -178,6 +184,67 @@ struct RuntimeData {
     config_dir: String,
 }
 
+impl RuntimeData {
+    fn state_file(&self) -> String {
+        format!("{}/state.txt", self.config_dir)
```

Comment:
The state file should not be in the config folder at all, it should be in somewhere like `~/.local/state/anyrun` or `~/.cache/anyrun`

Reply:
I would appreciate if this followed the XDG spec, e.g., `$XDG_CACHE_HOME` or `$XDG_STATE_HOME` by default

Reply:
Can I use dirs crate for this one and the config_dir ?